mlrun/feature_store/feature_vector.py

```diff
@@ -505,23 +505,19 @@ def status(self):
         """vector prep job status (ready, running, error)"""
         return self._merger.get_status()
 
-    def to_dataframe(self):
+    def to_dataframe(self, to_pandas=True):
```

Comment:
There are more options for dataframe than pandas yes or no, there is dask dataframe, spark, pandas, etc...
I guess this better be an enum rather than a bool

Reply:
@Hedingber we dont want to support all the options, pandas is the default common one to make things glueless so e.g. if u get feature vector and train w sklearn it always works regardless of the engine used, if a user wants something special e.g. map dask DF to spark he can turn the flag off and convert manually